mentoris/views.py

```diff
@@ -136,3 +136,25 @@ def request_translation(request, user_id):
         "notifications@kontinua.org",
         [email],
     )
+
+def main(request, volume_id = 1):
+    template = loader.get_template("mentapp/main.html")
+
+    volumes = Volume.objects.values_list('volume_id', flat=True).distinct().order_by('volume_id')
+
+    if volume_id is not None:
```

Comment:
Nit: This is just convention but instead of `if volume_id is not None: `use `if volume_id:`. It's the same logic but cleaner and more often found in industry coding practices